infra/resources/prod/westeurope/web_apps.tf

```diff
@@ -29,6 +29,18 @@ module "web_apps" {
     ]
   }
 
+  user_func = {
```

Comment:
`terraform plan` fails with

```tf
Error: Unsupported argument

  on web_apps.tf line 32, in module "web_apps":
  32:   user_func = {

An argument named "user_func" is not expected here.
Releasing state lock. This may take a few moments...
```

I think you should add the `user_func` variable to `web_apps` module